servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -194,7 +189,11 @@ private void handleMasterCommand(Command cmd) throws IOException {
       // Master requests blocks to be removed from Tachyon managed space.
       case Free:
         for (long block : cmd.mData) {
-          mBlockDataManager.removeBlock(Users.MASTER_COMMAND_USER_ID, block);
+          try {
+            mBlockDataManager.removeBlock(Users.MASTER_COMMAND_USER_ID, block);
+          } catch (NotFoundException nfe) {
+            throw new IOException(nfe);
```

Comment:
Tricky conversion --- `removeBlock` may return true `IOException` because it could not delete the file.